test/components/upgrades.test.js

```diff
@@ -178,7 +178,7 @@ describe('Upgrades testing', function () {
 
             await this.contracts.access.grantRole(this.roles.SCANNER_ADMIN, this.accounts.admin.address);
             expect(await scannerRegistry.getStakeController()).to.be.equal(this.contracts.stakingParameters.address);
-            expect(await scannerRegistry.version()).to.be.equal('0.1.3');
+            expect(await scannerRegistry.version()).to.be.equal('0.1.4');
```

Comment:
This seems wrong given that the test is called `0.1.0 -> 0.1.3`.

Reply:
The test name is wrong